src/arrays.js

```diff
@@ -1,62 +1,87 @@
 const getNthElement = (index, array) => {
-  // your code here
+  console.log(index % array.length);
+  return array[index % array.length];
 };
+//copied and pasted, couldn't work out alternative.
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
-};
+  const biggerArray = array.concat(element);
+  return biggerArray;
+};
+// {
+//   const biggerArray = array.slice()
+//   biggerArray.push(element);
+//   return biggerArray;
+// };
+//I took ages on this one, and came up with a longer solution before switching to concat - much simpler.
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index,1);
 };
 
-const numbersToStrings = numbers => {
-  // your code here
+function numbersToStrings(numbers) {
+  return numbers.toString().split(',');
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
-};
+  const upperCaseStrings = strings.map(a => a.toUpperCase());
+  return upperCaseStrings;
+};
+function reverseWordsInArray(strings) {
```

Comment:
Always good to leave a clear line after every function to make reading easier
```suggestion
};

function reverseWordsInArray(strings) {
```